packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -317,6 +318,7 @@ export const transformContentTypesToModels = (
       },
       indexes: contentType.indexes as Model['indexes'],
       foreignKeys: contentType.foreignKeys as Model['foreignKeys'],
+      lifecycles: contentType?.lifecycles,
```

Comment:
```suggestion
      lifecycles: contentType?.lifecycles ?? {}, 
```
Since we define lifecycles as required (and it was required in v4 too apparently)